python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -261,7 +390,9 @@ def _throughput_calculate_helper(iterrun, func, *args):
         st = time.time()
         func(*args)
         time_list.append(time.time() - st)
-    # TODO: remove first and last 10 percent
+    time_list.sort()
```

Comment:
I think we don't need to sort here. if we sort this time list, then removing the top/least 10% data is meaningless, we will always find the median of the whole time list anyway.

Reply:
changed to np.mean. thanks.